src/EFCore.PG.Trgm/Infrastructure/Internal/NpgsqlTrgmOptionsExtension.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.Extensions.DependencyInjection;
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Query;
+
+using Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators.Internal;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure.Internal
+{
+    public class NpgsqlTrgmOptionsExtension : IDbContextOptionsExtension
+    {
+        public DbContextOptionsExtensionInfo Info => new ExtensionInfo(this);
+
+        public virtual void ApplyServices(IServiceCollection services)
+        {
+            services.AddEntityFrameworkNpgsqlTrgm();
+        }
```

Comment:
nit: No need in braces since it's an one liner (:

Reply:
24ff45d1edc00f0f4748be34ee18e0c0eee25f3e fixes this.